### PR TITLE
B2 b search field

### PR DESCRIPTION
**Pull request by ArpithaNagappa**
Added search feature to search  books by author/title.

### CODE REVIEW

File: src/components/Home.jsx [hunk 1]
```diff
@@ -1,12 +1,14 @@
 import React from 'react'
 import Navigation from './Navigation'
 import SearchBar from './SearchBar'
+import DiplayBooks from './DiplayBooks'
 
 const Home = () => {
   return (
     <div>
         <Navigation/>
-        <SearchBar/>
```
Comment (jnorrick) [inline src/components/Home.jsx:9]: just to double check, was SearchBar intended to be removed here?

**Review by jnorrick — COMMENTED** [review]
I'm getting the following error: 
image
I also left a comment re: intention to remove SearchBar in some of the code.

File: src/components/Home.jsx (hunk 1 above)
Comment (ArpithaNagappa) [reply src/components/Home.jsx:9]: @jnorrick  I am currently not using SearchBar component so, I removed it in Home.jsx.

**Comment by MatthewSpeaks** [comment]
Hi Arpitha, 

I think overall this looks good. But in your DisplayBooks.jsx file, we need to change where you are fetching from to 'http://localhost:8080/api/books/home' and it works.

**Comment by ArpithaNagappa** [comment]
Sorry, I forgot to push home Controller code, I just updated that, please check it out.

**Review by dalpiad — COMMENTED** [review]
Arpitha,
Your code will work but isn't DRY.  I recommend just changing the fetch location to what Matt suggested.  Then the change to the home controller won't be needed.  However, this is just a suggestion if you want to continue with the current changes I will "approve" and merge.

File: src/components/DiplayBooks.jsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    const [books, setBooks] = useState([]);
+    const [records, setRecords]=useState(books);
+    const Filter= (event)=>{
+        setRecords(books.filter(f=>(f.title.toLowerCase().includes(event.target.value.toLowerCase()) 
+        ||(f.author.toLowerCase().includes(event.target.value.toLowerCase()) ) )))
+    }
+    
+      
+    useEffect(() => {
+        fetch('http://localhost:8080/Home')
```
Comment (dalpiad) [inline src/components/DiplayBooks.jsx:20]: replace the fetch URL with 'http://localhost:8080/api/books/home'

Comment (ArpithaNagappa) [reply src/components/DiplayBooks.jsx:20]: Make sense, I will update that in Display.jsx file, thanks!

Comment (ArpithaNagappa) [reply src/components/DiplayBooks.jsx:20]: Can anyone of you please reject my front-home branch merge request, thanks!

**Pull request closed without merging**